_posts/2018-07-25-Relay-Networking-Deep-Dive.markdown

```diff
@@ -220,8 +265,8 @@ const PostFragment = createFragmentContainer(Post, {
 });
 ```
 
-In the fragment above, it will first get title and commentsCount data from Post, and after it will get the data for
-CommentsList_post fragment.
+In the fragment above, Relay will first get the `title` and `commentsCount` from the Post, then afterwards Relay
+will get the data for `CommentsList_post` fragment. Sending both through the observable.
```

Comment:
My rule of thumb for writing is to first write "it" in a sentence, then come back and re-add the thing that is being referred to. It's a bit more verbose, but much clearer to read.